src/17D6A0.c

```diff
@@ -2246,11 +2246,11 @@ s32 bMarioDefendAnims[] = {
     STATUS_FROZEN,          ANIM_Mario1_Frozen,
     STATUS_TRANSPARENT,     ANIM_Mario1_Idle,
     STATUS_TURN_DONE,       ANIM_Mario1_Still,
-    STATUS_13,              ANIM_MarioB2_BerserkStill,
+    STATUS_TURN_DONE_BERSERK,   ANIM_MarioB2_BerserkStill,
```

Comment:
This breaks the table alignment :/

Reply:
Need more input, looks like both marjin and I would prefer that we stop doing alignment in these files, as it makes for bad diffs ([relevant Discord discussion](https://discord.com/channels/279322074412089344/768585278331289622/1108316885503721524)). Therefore, if @ethteck and @HailSanta agree, we can just leave this line as-is and remove the clang-format rule that enforces alignment (bearing in mind use of clang-format is optional for contributions right now)

Reply:
I think we should continue doing alignment in the base repo, but not enforce it for mods.